dCommon/dClient/AssetManager.cpp

```diff
@@ -92,17 +94,19 @@ bool AssetManager::GetFile(const char* name, char** data, uint32_t* len) {
 	auto fixedName = std::string(name);
 	std::transform(fixedName.begin(), fixedName.end(), fixedName.begin(), [](uint8_t c) { return std::tolower(c); });
 	std::replace(fixedName.begin(), fixedName.end(), '\\', '/'); // On the off chance someone has the wrong slashes, force forward slashes
-	auto realPathName = fixedName;
 
-	if (std::filesystem::exists(m_ResPath / realPathName)) {
+	// Special case for unpacked client have BrickModels in upper case
+	if (this->m_AssetBundleType == eAssetBundleType::Unpacked) GeneralUtils::ReplaceInString(fixedName, "brickmodels", "BrickModels");
+
+	if (std::filesystem::exists(m_ResPath / fixedName)) {
 		FILE* file;
 #ifdef _WIN32
```

Comment:
I would prefer to go an abstraction level up to ifsteams here, avoids the ugly preprocessing. Doesn't require changing, just a note.

Reply:
I was doing this for speed reasons, no idea how much it actually matters.